lib/i18n/payment-method-provider-type.js

```diff
@@ -32,6 +32,8 @@ const i18nTypes = defineMessages({
 export const i18nPaymentMethodProviderType = (intl, type) => {
   if (type === GQLV2_PAYMENT_METHOD_TYPES.PAYPAL) {
     return 'PayPal';
+  } else if (type === GQLV2_PAYMENT_METHOD_TYPES.CRYPTO) {
+    return 'Crypto Contribution';
```

Comment:
Where is this appearing? No translation?

Reply:
Yes you are right, no need to seperate `else if` condition here. We can add it to the translations list. Done. 👍🏽